android/modules/geolocation/src/java/ti/modules/titanium/geolocation/GeolocationModule.java

```diff
@@ -786,14 +786,19 @@ private boolean shouldUseUpdate(Location newLocation)
 	 * @return						map of property names and values that contain information
 	 * 								pulled from the specified location
 	 */
+	@SuppressWarnings("NewApi")
 	private KrollDict buildLocationEvent(Location location, LocationProvider locationProvider)
 	{
 		KrollDict coordinates = new KrollDict();
 		coordinates.put(TiC.PROPERTY_LATITUDE, location.getLatitude());
 		coordinates.put(TiC.PROPERTY_LONGITUDE, location.getLongitude());
 		coordinates.put(TiC.PROPERTY_ALTITUDE, location.getAltitude());
 		coordinates.put(TiC.PROPERTY_ACCURACY, location.getAccuracy());
-		coordinates.put(TiC.PROPERTY_ALTITUDE_ACCURACY, null); // Not provided
+		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```

Comment:
Personally I'd prefer to see the actual api version it is supported from. I'm not sure too many people are familiar with the "brand name" to api level mappings.

Reply:
I think the API version is less important here, it's nice to see what Android version supports altitude accuracy (i.e most people know Android Oreo is 8.0, where `26` would be less obvious).